public/js/extra.js

```diff
@@ -867,7 +867,11 @@ const linkifyAnchors = (level, containingElement) => {
     if (header.getElementsByClassName('anchor').length === 0) {
       if (typeof header.id === 'undefined' || header.id === '') {
         // to escape characters not allow in css and humanize
-        const id = slugifyWithUTF8(getHeaderContent(header))
+        var id = slugifyWithUTF8(getHeaderContent(header))
+        // to make compatible with GitHub, GitLab, Pandoc and many more
+        if (window.linkifyHeaderStyle !== 'keep-case') {
```

Comment:
```suggestion
        if (window.linkifyHeaderStyle === 'gfm') {
```

Or any `linkifyHeaderStyle` value except `keep-case` would go into here (maybe that's your intent?)

Reply:
yes that was my intend, because everywhere else it is lower-case. the only reason to not move to lower-case as default - in my eyes - is backwards compatibility.

Reply:
I imagine people using any kind of Markdown flavor as value in this variable; examples from Pandoc:

* _markdown_ (Pandoc's Markdown)
* _markdown\_github_ (or _gfm_)
* _markdown\_mmd_
* _markdown\_phpextra_
* _markdown\_strict_

I was also already thinking about the possibility to one day choose the Markdown flavor for everything (not just the reference style). In that case, it makes sense to already have this feature designed in a somewhat compatible fashion, not requiring yet more backwards-compatibility precautions, for example.